magstock/templates/static_views/legal_name.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
The problem with this approach is that we've basically copied an entire file into a second repo to change a single word.

A better approach would probably be to change the wording in the original repo so that instead of saying "upon check-in to claim their wristband" or "upon check-in to claim their wristband" it can just say "upon check-in at our registration desk."

That way we don't need to copy this whole file, which is problematic because if we want to tweak the wording elsewhere we'd need to update it in two places.